src/workerd/api/hyperdrive.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+
+#include "streams.h"
+#include <capnp/compat/json.h>
+#include <cstdint>
+#include <kj/common.h>
+#include <kj/async-io.h>
+#include <workerd/jsg/jsg.h>
+#include <workerd/util/http-util.h>
+
+namespace workerd::api::public_beta {
```

Comment:
Is the `:public_beta` part of the namespace here necessary?

Reply:
No, not at all, I can remove it